pylabrobot/liquid_handling/liquid_handler_tests.py

```diff
@@ -249,6 +254,50 @@ async def test_move_plate_onto_resource_stack_with_plate(self):
     self.assertEqual(plate2.location.z, 15)
     self.assertEqual(stack.get_absolute_size_z(), 30)
 
+  async def test_move_plate_rotation(self):
+    rotations = [0, 90, 270, 360]
+    grip_directions = [
+        (GripDirection.LEFT, GripDirection.RIGHT),
+        (GripDirection.FRONT, GripDirection.BACK),
+    ]
+    site_classes = [ResourceStack, PlateCarrierSite]
```

Comment:
anything else I should put in here?

Reply:
Yes. I like that.

Reply:
Will do it. Anything else you can think of? Tilter?

Reply:
tilter, heater shaker, shaker, temperature controllers, all these classes. potentially a lot easier to just create a new class SinglePlateTakingResource, override assign_child_resource there, and then have these on-deck/off-deck devices inherit from that

Reply:
Yea agreed